metabase/metabase
# fix cypress.js included twice

Closes https://github.com/metabase/metabase/issues/22712

## Changes

Code in `cypress.js` was twice because we imported it in almost each test since it reexports helpers. Seems like Cypress creates two different bundles, ones of which is for tests code. The solution is to reexeport helpers from another file.

## How to verify

1. open `https://github.com/metabase/metabase/blob/master/frontend/test/__support__/e2e/cypress.js`
2. Anywhere in that file write `console.log("foo");`
3. Run any of the Cypress tests in the open mode and observe the console
4. "foo" is printed once
